[2023-09-22T17:25:53.878+0700] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crawl_dag_v01.run_my_async_task manual__2023-09-22T09:05:17.387394+00:00 [queued]>
[2023-09-22T17:25:53.884+0700] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crawl_dag_v01.run_my_async_task manual__2023-09-22T09:05:17.387394+00:00 [queued]>
[2023-09-22T17:25:53.885+0700] {taskinstance.py:1359} INFO - Starting attempt 6 of 11
[2023-09-22T17:25:53.895+0700] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): run_my_async_task> on 2023-09-22 09:05:17.387394+00:00
[2023-09-22T17:25:53.900+0700] {standard_task_runner.py:57} INFO - Started process 1862 to run task
[2023-09-22T17:25:53.904+0700] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crawl_dag_v01', 'run_my_async_task', 'manual__2023-09-22T09:05:17.387394+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/crawlInvest.py', '--cfg-path', '/tmp/tmp_ljkhwvy']
[2023-09-22T17:25:53.905+0700] {standard_task_runner.py:85} INFO - Job 69: Subtask run_my_async_task
[2023-09-22T17:25:53.954+0700] {task_command.py:415} INFO - Running <TaskInstance: crawl_dag_v01.run_my_async_task manual__2023-09-22T09:05:17.387394+00:00 [running]> on host 92f43f7b42a7
[2023-09-22T17:25:54.017+0700] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='linhhv' AIRFLOW_CTX_DAG_ID='crawl_dag_v01' AIRFLOW_CTX_TASK_ID='run_my_async_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-22T09:05:17.387394+00:00' AIRFLOW_CTX_TRY_NUMBER='6' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-22T09:05:17.387394+00:00'
[2023-09-22T17:25:54.031+0700] {logging_mixin.py:151} INFO - 1
[2023-09-22T17:25:54.036+0700] {logging_mixin.py:151} WARNING -   0%|          | 0/1 [00:00<?, ?it/s]
[2023-09-22T17:25:55.618+0700] {logging_mixin.py:151} WARNING -   0%|          | 0/1 [00:01<?, ?it/s]
[2023-09-22T17:25:55.619+0700] {logging_mixin.py:151} INFO - 'list' object is not callable
[2023-09-22T17:25:55.620+0700] {python.py:194} INFO - Done. Returned value was: None
[2023-09-22T17:25:55.631+0700] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=crawl_dag_v01, task_id=run_my_async_task, execution_date=20230922T090517, start_date=20230922T102553, end_date=20230922T102555
[2023-09-22T17:25:55.667+0700] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-22T17:25:55.689+0700] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
